rero_ils/modules/documents/serializers/ris.py

```diff
@@ -24,11 +24,14 @@
 
 from rero_ils.modules.commons.identifiers import IdentifierFactory, \
     IdentifierType
+from rero_ils.modules.documents.api import Document
+from rero_ils.modules.documents.serializers.base import \
+    BaseDocumentFormatterMixin
+from rero_ils.modules.documents.utils import process_literal_contributions
 from rero_ils.utils import get_i18n_supported_languages
 
 from .base import BaseDocumentFormatterMixin
 from ..api import Document
```

Comment:
Must be remove because you already add them using fullpath above

Reply:
I think we need to keep relative import instead absolute import because we are in the same module.